lib/mix/tasks/header_footer.ex

```diff
@@ -133,6 +135,21 @@ if Mix.env() in [:dev, :test] do
       {:ok, _files} = File.rm_rf("export")
     end
 
+    defp files_list_reducer(filename, path, base_path, acc) do
```

Comment:
This function is doing a lot. It's hard to read. It can be broken out further to be more clear.